### PR TITLE
Add NPM script for renaming project and creating a Github repo

### PR DESCRIPTION
**Pull request by andrewgrewell**
I like the idea of the clean script, this just takes the concept a little further and adds a script that will walk the client through renaming their project and automatically creating a github repo.

### CODE REVIEW

**Comment by KarandikarMihir** [comment]
@andrewgrewell Thanks for submitting this PR. May I know which issue does this PR point to?

**Comment by KarandikarMihir** [comment]
@andrewgrewell I'm sorry I have to close this. The changes look intriguing to me. But please follow [contribution guidelines](https://github.com/mxstbr/react-boilerplate/blob/master/.github/CONTRIBUTING.md) before submitting a PR. Please create an issue, let the community have their say first. Thanks!

**Comment by andrewgrewell** [comment]
@KarandikarMihir 
Sorry, thanks for the heads up

File: internals/scripts/init-github-project.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  }, {
+    type: 'input',
+    name: 'description',
+    message: 'What is your project description?',
+    default: 'a project created with react-boilerplate',
+    validate: (value) => /.+/.test(value) ? true : 'The description is required'
+  }, {
+    type: 'input',
+    name: 'author',
+    message: 'Who is your the project author?',
```
Comment (spen) [inline internals/scripts/init-github-project.js]: A little nitpick with the wording here: 'Who is your the project author?' could be 'Who is the project author?' or 'What is the name of the project author?'

Comment (andrewgrewell) [inline internals/scripts/init-github-project.js]: Good catch

**Pull request closed without merging**